test/quicer_snb_SUITE.erl

```diff
@@ -248,7 +248,7 @@ tc_stream_owner_down(Config) ->
                | default_stream_opts() ],
   Options = {ListenerOpts, ConnectionOpts, StreamOpts},
   ct:pal("Listener Options: ~p", [Options]),
-  ?check_trace(#{timetrap => 1000},
+  ?check_trace(#{timetrap => 10000},
```

Comment:
Maybe timeouts can be removed from block_until, since this testcase already has a timetrap to minimize flakiness?

Reply:
It could be the server client connect/sending message parts  are  broken, I hope we could have a default timeout to not make test run too long in case of errors.